src/transformers/modeling_utils.py

```diff
@@ -2997,7 +2989,8 @@ def from_pretrained(
 
         # Check first if we are `from_pt`
         if use_keep_in_fp32_modules:
-            low_cpu_mem_usage = True
+            if is_accelerate_available():
+                low_cpu_mem_usage = True
```

Comment:
Reading this, it seems that we don't necessarily have to use `low_cpu_mem_usage` no? 
(As previously we never went through that path when accelerate was not installed)

Reply:
I have no idea why there was

```python
        if use_keep_in_fp32_modules:
            low_cpu_mem_usage = True
```

in the first place, maybe @younesbelkada knows